IPs/RSKIP_tbd.md

```diff
@@ -0,0 +1,229 @@
+
+
+## Motivation
+
+The Powpeg election is a critical process that should have as many reassurances as possible.
+As mentioned in the previous section, nowadays once the election takes place, the process is irreversible.
+This RSKIP proposes an automated and decentralized process to add one more layer of security to this process to avoid any human mistake putting funds at risk.
+
+This process protects funds from:
+- An invalid redeemscript that would render the funds burned.
+- A composition of pegnatories that are not responsive and, therefore, unable to sign transactions.
+
+
+## Specification
+
+This RSKIP proposes changes in the Powpeg change process. The subsections will expand on the changes for each step.
+
+The current process consists in the following phases:
+1. Election start: m/n authorized members propose to start a new election.
+2. Composition proposal: m/n vote for the new composition.
+3. New composition commitment: m/n vote to commit new composition. New elections are disabled.
+4. Pre activation phase: immediately after commitment and during a number of blocks. No real change for end users.
+5. Activation phase: new Powpeg address activates, two-way peg starts operating from this new address.
+6. Migration phase: Funds from the retiring Powpeg wallet are moved to the active Powpeg.
+7. Post migration phase: Retiring Powpeg information is removed from the Bridge. New elections are re-enabled.
+
+The election process in itself doesn't change: the authorized members will still propose a new composition and once the voting finishes the commit event will be emitted, but the proposed Powpeg will not be committed yet and its validation will start.
+
+
+### New phase - Validation
+
+A new phase will be added, in between the _composition proposal_ and _new composition commitment_ phases, the _validation_ phase.
+During a certain number of blocks a new protocol will be started, Spendability Validation Protocol (SVP).
+
+If this protocol succeeds, the Bridge will progress to the next phase (_new composition commitment_).
+If this protocol fails, things will return to their previous state.
+
+
+### Spendability Validation Protocol (SVP)
+
+The Spendability Validation Protocol is a new process in which the current Powpeg will send funds to the newly elected Powpeg, for it to send them back, proving that it can truly hold and spend funds, before proceeding with the activation and migration of the funds in the wallet.
+
+This protocol involves three parts:
+- Proposed Powpeg funding: the proposed Powpeg won't have funds to create the proof so the current Powpeg will create a UTXO covering the expenses.
+- Proof transaction creation: once the proposed Powpeg receives the funds, it will create a UTXO to the current Powpeg to proof that it can spend them.
```

Comment:
```suggestion
- Proof transaction creation: once the proposed Powpeg receives the funds, it will create an UTXO to the current Powpeg to proof that it can spend them.
```